lib/opentracing/reference.rb

```diff
@@ -24,10 +24,7 @@ class Reference
     #   ])
     #
     def self.child_of(context)
-      if context.is_a?(Span)
-        context = context.context
-      end
-
+      context = context.context if context.is_a?(Span)
```

Comment:
Couldn't this method just be one line:

```ruby
Reference.new(CHILD_OF, context.try(:context))
```

Reply:
No. `try` is rails specific method. And even then I think it returns `nil` if there's no such method.

Reply:
![image](https://user-images.githubusercontent.com/7880/35102106-2769ef3e-fc1f-11e7-92da-95ef7e4818c1.png)